### ISSUE TITLE
First Android app codelab: 

### ISSUE THREAD

**Issue by reknoZ** [google-developer-training/first-android-app#7]
**Describe the problem**
The floating action button is visible at the beginning but then it's gone. But then it's back. And then it's gone. Very weird and disconcerting especially when following the tutorial and things just don't look the same as the tutorial shows.

**In which step of the codelab can this issue be found?**
Many steps. Task 4, Step 5 - floating action button is present. Task 5, all steps - it's gone. Task 6 - it's back at the beginning, but gone in Step 1. But it's back in Step 7. Etc.

**Versions**
1. What version of Android Studio are you using? 3.6 beta 5
2. What API level are you targeting? doesn't matter.

**Additional information**
Your code labs are a great idea. But the implementation is pretty unimpressive. You have lots and lots of descriptive text, mixed in with instructions for a reader to follow. The descriptive text should be distinctly different from the steps to follow. And each step should include just *one* thing to do, not two or three or four things.

That, combined with the images that are inconsistent, makes it very hard to follow.

I need a job, so if you need someone to refactor some Codelabs, I'd be happy to go through them, rewrite them and take screen shots. This is not a joke. I need a job.

**codelab:** build-your-first-android-app-kotlin

**Comment by GeneAndGoogle** [comment]
Some images (i.e., those with an app bar, etc.) are screenshots of the app as it runs on the emulator or a device. Other images show how the app looks in the layout editor. There's no appear and disappear -- the images are showing different things.
